README.md

```diff
@@ -21,27 +21,36 @@ here and there, but I think it is working.
 The code is now it's beta phase. See the implemented column for what has
 already been implemented.
 
-Dependencies:
--------------
+## Dependencies:
+
 
 Python packages required: See the environment.yml file
 
 To use the movie saving feature: ffmpeg & xterm.
 
 Iseult should work on Windows, MacOS & Linux.
 
-To run Iseult on tigressdata or stellar vis nodes type the following:
+## Setup & Running
+
+There are two different methods to setup what you need for Iseult to run. The recommended method is to setup a python environment with conda, pyenv, etc and a conda `environment.yml` file is provided. The other method for those with access to the Stellar cluster at Princeton is to simply load the proper Anaconda module.
 ```bash
-# Initial setup, optional if your python environment has the correct version of all dependencies
+# Recommended setup
+# If on Stellar
 $ module load anaconda3/2024.6 # or the latest version
+# If not on Stellar then install/load Anaconda
 $ cd /path/to/Iseult/
-$ chmod +x ./iseult.py
 $ conda env create
 $ conda activate iseult
+$ chmod +x ./iseult.py
+
+# Stellar simple setup
+$ module load anaconda3/2023.3
+$ cd /path/to/Iseult/
+$ chmod +x ./iseult.py
 
 # After initial setup
-$ module load anaconda3/2024.6 # or the latest version
-$ conda activate iseult
+# make sure the proper environment is activated or load the proper Anaconda
```

Comment:
how do I make sure that the proper environment is activated? please list the command I need to run

Reply:
I added instructions